builder/tests/test_views.py

```diff
@@ -184,6 +185,9 @@ def test_discard_only_draft_version(client, organisation_user):
         author=organisation_user,
         name="foo",
         coding_system_id="snomedct",
+        coding_system_database_alias=CODING_SYSTEMS["snomedct"]
+        .most_recent()
+        .database_alias,
```

Comment:
This pattern comes up a few times.  Perhaps there could be a helper function (eg `coding_systems.most_recent_database_alias("snomedct")`)?

Reply:
I've done that now - hadn't realised quite how many times it was used :)